code/drasil-gool/lib/Drasil/Shared/RendererClassesCommon.hs

```diff
@@ -69,18 +70,23 @@ class BodyElim r where
 class RenderBlock r where
   multiBlock :: [MSBlock r] -> MSBlock r
 
+-- This assumes that the renderers encode Body as Doc
```

Comment:
Right, this assumes that the only thing to ever be done with a `Block` will be to render it. This might not be a good assumption.